build/lib/buildkit.sh

```diff
@@ -34,25 +87,32 @@ fi
 if [ -f "/buildkit.sh" ]; then
     for i in $(seq 1 5); do
         [ $i -gt 1 ] && sleep 15
-        $CMD "$@" && s=0 && break || s=$?
+        $CMD $ARGS && s=0 && break || s=$?
     done
 
     # space is limited on presubmit nodes, after each image build clear the build cache
     if [ "${JOB_TYPE:-}" == "presubmit" ] && [ "${PRUNE_BUILDCTL:-false}" == "true" ]; then
-        $CMD prune --all
+        buildctl prune --all
     fi
 
     (exit $s)
 else
     # skip retry when running locally
     log_file=$(mktemp)
     trap "rm -f $log_file" EXIT
-    if ! $CMD "$@" 2>&1 | tee $log_file; then        
+    if ! $CMD $ARGS 2>&1 | tee $log_file; then      
         if grep -q "blobs/uploads/\": EOF" $log_file ; then
             echo "******************************************************"
             echo "Ensure container registry and repository exists!!"
             echo "Try running make create-ecr-repos to create ecr repositories in your aws account."
             echo "******************************************************"
+        elif [ "$CMD" = "docker buildx" ] && grep -q "multiple platforms feature is currently not supported" $log_file; then
```

Comment:
this is a bit annoying but the default docker buildx builder does not support multiarch builds. If people are building just single arches this isnt an issue, but added this message check to guide them on how to create a driver that will support multiple.  i tested this on an al2 instance.